Mobile/stockdogmobile/screens/portfolioOld.js

```diff
@@ -12,9 +12,8 @@ import PortfolioItem from '../components/portfolioItem';
 import Api from '../api';
 import LoadingProfile from '../components/loadingProfile';
 import NavBar from '../components/navbar';
-import Drawer from 'react-native-drawer';
 
-export default class Profile extends Component {
+export default class PortfolioOld extends Component {
```

Comment:
It might be a good idea just to keep this old file on your computer and remove it from the repo to prevent confusion.